ERCS/erc-7715.md

```diff
@@ -87,23 +111,40 @@ Request example:
     type: 'key',
     data: {
       id:'did:key:zQ3shokFTS3brHcDQrn82RUDfCZESWL1ZdCEJwekUDPQiYBme',
-      }
     }
+  }
   permissions: [
     {
-      type: 'erc20-spending-limit',
-      data: {
-        erc20Address: '0x...',
-        limit: '0x...',
-      },
-      required: true
-    },
-    {
-      type: 'gas-limit',
+      type: 'erc20-transfer',
+      policies: [
+        {
+          type: 'token-allowance',
+          properties: {
+            allowance: 'uint256'
```

Comment:
If the properties of a `type` are strongly typed and enforced by the wallet, then why does the requestor need to declare the types?